src/set/bytes.rs

```diff
@@ -0,0 +1,130 @@
+//! Serializer codec for serializing a list of byte arrays as a set
+//!
+//! # Usage
+//!
+//! To use, annotate the field with `#[serde(with = "serde_dynamo::set::bytes")]`.
+//!
+//! DynamoDB will return an error if given an empty set. Thus, it may
+//! be beneficial to additionally annotate the field with `#[serde(default)]`
+//! and `#[serde(skip_serializing_if = "<empty check>")]`. This will make sure
+//! that the field is omitted when empty.
+//!
+//! This serializer does not check for duplicate values or an empty set.
+//! If the set contains duplicate values or is empty, DynamoDB will return a
+//! validation error when the attribute value is used.
+//!
+//! # Errors
+//!
+//! The serializer in this module will return an error if:
+//!
+//! * the value does not serialize as a sequence
+//! * the sequence contains any value that is not a byte array
+//!
+//! # Examples
+//!
+//! ```
+//! use serde_bytes::ByteBuf;
+//! use serde_derive::{Serialize, Deserialize};
+//! use serde_dynamo::{Item, AttributeValue};
+//!
+//! #[derive(Serialize, Deserialize)]
+//! struct MyStruct {
+//!     #[serde(with = "serde_dynamo::set::bytes")]
```

Comment:
I feel like

```rust
#[serde(with = "serde_dynamo::byte_set")]
#[serde(with = "serde_dynamo::number_set")]
#[serde(with = "serde_dynamo::string_set")]
```

Reply:
I can made that change; this was mainly just for organization and avoiding a bunch of modules at the crate base. I've been using _bytes set_ (with the plural) since it's a set of byte arrays. If that's okay, I can rename and move these modules into the crate root.